unison-src/transcripts/errors/ucm-hide-all-error.md

```diff
@@ -8,5 +8,5 @@ then the transcript parser should print the stanza
 and surface a helpful message.
 
 ```ucm:hide:all:error
-.> ls
+.> history
```

Comment:
@aryairani came back to this with a clear head, and managed to debug it. `ls` was returning an error, so both of these were actually interpreted as "an error occurs in a block where an error is expected."